kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java

```diff
@@ -1051,6 +1052,32 @@ public void testQueryDataServiceUsingComplexCustomQueryBuilderFilterSpecWithOrde
 
 
     }
+    
+    @Test
+    public void testDefaultQueryJbpmProcessInstances() throws Exception {
```

Comment:
Doesn't really throw any exception inside this method.

```suggestion
    public void testDefaultQueryJbpmProcessInstances() {
```

Reply:
done